cli/tsc/99_main_compiler.js

```diff
@@ -12,7 +12,7 @@
 // https://tc39.es/ecma262/#sec-get-object.prototype.__proto__
 delete Object.prototype.__proto__;
 
-((window) => {
+((/** @type {any} */ window) => {
```

Comment:
Typing this `/** @` part caused the lsp to hang, which sent me down a rabbit hole all day trying to figure out what was up. There was an infinite loop in TypeScript and I reported it in their repo.